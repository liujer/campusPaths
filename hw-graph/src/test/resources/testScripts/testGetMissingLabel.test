# Tests the method getLabel in DLGraph by adding nodes
# and edges to a graph and checking if the label returned
# is correct
CreateGraph A
AddNode A one
AddNode A two
AddNode A three
AddEdge A one two e12
AddEdge A one three e13
GetLabel A two three
GetLabel A one Two
