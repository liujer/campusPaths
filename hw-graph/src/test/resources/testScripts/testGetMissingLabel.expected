# Tests the method getLabel in DLGraph by adding nodes
# and edges to a graph and checking if the label returned
# is correct
Created graph A
Added node one to A
Added node two to A
Added node three to A
Added edge e12 to A
Added edge e13 to A
Label of two and three does not exist in A
Label of one and Two does not exist in A
